stablehlo/reference/ProcessGrid.h

```diff
@@ -51,10 +51,12 @@ struct ProcessId {
 };
 
 // StableHLO `process_group`.
-class ProcessGroup : public SmallVector<ProcessId> {};
+struct ProcessGroup : public SmallVector<ProcessId> {};
 
 // StableHLO `process_groups`.
-class ProcessGroups : public SmallVector<ProcessGroup> {};
+struct ProcessGroups : public SmallVector<ProcessGroup> {
+  ProcessGroups findAll(ProcessId processId);
```

Comment:
I don't think that `findAll` is the right API to express what we're looking for here, especially since the current caller simply does `[0]` on the result. Something like `std::optional<ProcessGroup> find(...)` looks better.